src/prewrap/bitmasks.jl

```diff
@@ -26,18 +26,22 @@ end
 (&)(a::BitMask, b::BitMask) = error("Bitwise operation not allowed between incompatible bitmasks '$(typeof(a))', '$(typeof(b))'")
 (|)(a::BitMask, b::BitMask) = error("Bitwise operation not allowed between incompatible bitmasks '$(typeof(a))', '$(typeof(b))'")
 xor(a::BitMask, b::BitMask) = error("Bitwise operation not allowed between incompatible bitmasks '$(typeof(a))', '$(typeof(b))'")
+isless(a::BitMask, b::BitMask) = error("Bitwise operation not allowed between incompatible bitmasks '$(typeof(a))', '$(typeof(b))'")
 
 (&)(a::T, b::T) where {T <: BitMask} = T(a.val & b.val)
 (|)(a::T, b::T) where {T <: BitMask} = T(a.val | b.val)
 xor(a::T, b::T) where {T <: BitMask} = T(xor(a.val, b.val))
+isless(a::T, b::T) where {T <: BitMask} = T(isless(a.val, b.val))
 
 (&)(a::T, b::Integer) where {T <: BitMask} = T(a.val & b)
 (|)(a::T, b::Integer) where {T <: BitMask} = T(a.val | b)
 xor(a::T, b::Integer) where {T <: BitMask} = T(xor(a.val, b))
+isless(a::T, b::Integer) where {T <: BitMask} = T(isless(a.val, b))
 
-(&)(a::Integer, b::T) where {T <: BitMask} = b & a
-(|)(a::Integer, b::T) where {T <: BitMask} = b | a
-xor(a::Integer, b::T) where {T <: BitMask} = xor(b, a)
+(&)(a::Integer, b::T) where {T <: BitMask} = b.val & a
+(|)(a::Integer, b::T) where {T <: BitMask} = b.val | a
+xor(a::Integer, b::T) where {T <: BitMask} = xor(b.val, a)
+isless(a::Integer, b::T) where {T <: BitMask} = isless(b.val, a)
```

Comment:
For these lines you don't need to add `.val`, since it falls back to the methods above.

Reply:
Ah, I see! Yes, I missed that. I'll remove it.